activemodel/test/cases/validations/numericality_validation_test.rb

```diff
@@ -237,6 +237,13 @@ def test_validates_numericality_with_in
     assert_valid_values([1, 2, 3])
   end
 
+  def test_validates_numericality_with_in_array
+    Topic.validates_numericality_of :approved, in: [1, 3]
```

Comment:
Theoretically, this should work with an `Enumerable` too, yes?

```suggestion
  def test_validates_numericality_with_in_enumerable
    Topic.validates_numericality_of :approved, in: [1, 3].each
```